### ISSUE TITLE
dingo_tests

### ISSUE THREAD

**Issue by dastoqc** [dingo-cpr/dingo_robot#27]
Hi.

RosDep seems to fail on finding 'dingo_tests' package which is set as a dependency in dingo_robot package.xml. Commenting it seems to be a harmless fix for now in my setup.

Best regards,
David.

**Comment by tonybaltovski** [comment]
Hi,

This package is bloomed on Clearpath's buildfarm.  To get packages and dependencies from Clearpath's buildfarm, follow this: https://wiki.ros.org/ClearpathRobotics/Packages

**Comment by dastoqc** [comment]
Thank you for your quick answer. ROSDep still gives me `dingo_robot: Cannot locate rosdep definition for [dingo_tests]`

**Comment by tonybaltovski** [comment]
Can you install it manually using apt?

**Comment by dastoqc** [comment]
Yes, that's a working solution, but I'd prefer to let rosdep deal with it.

**Comment by tonybaltovski** [comment]
Can you post the output of when you run `rosdep update`?
